Resources/translations/sulu/backend.en.xlf

```diff
@@ -187,6 +191,10 @@
                 <source>l91_sulu_form.choice</source>
                 <target>Choices</target>
             </trans-unit>
+            <trans-unit id="64">
+                <source>l91_sulu_form.success_text</source>
+                <target>Success message</target>
+            </trans-unit>
```

Comment:
@thomasduenser translation for key "l91_sulu_form.website_configuration" is missing for "en" and "fr"